docs/reference/announcements-release-notes/870/870-release-notes.md

```diff
@@ -19,6 +19,83 @@ These release notes identify the new features included in 8.7, including [alpha
 | ---------------------- | ---------------------------- | ------------ | ------------ | ------------ |
 | 8 April 2025           | 13 October 2026              | -            | -            | -            |
 
+<!-- https://github.com/camunda/product-hub/issues/1063 -->
+
+### Cluster connector management in Console <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span><span class="badge badge--medium" title="This feature affects Console">Console</span> {#manage-connectors}
+
+Cluster connector management allows you to monitor and manage your running inbound connector [webhooks, message queue subscriptions, and polling subscriptions](/reference/glossary.md#inbound-connector).
+
+- To open the **Connector Management** page, on the cluster **Overview** tab, click **Manage** on the Connectors component tile.
+- Use this feature to check your inbound connectors are healthy and running, and troubleshoot unhealthy connectors.
+- For example, you can see if a connector instance is unhealthy, and use the activity log to troubleshoot and resolve issues.
+
+<!-- To learn more about this feature, see [manage your connectors](/components/console/manage-clusters/manage-connectors.md). -->
+
+<!-- https://github.com/camunda/product-hub/issues/2521 -->
+
```

Comment:
Here are some platform changes for the Web Modeler side

Non-public Database Schema Support (SM only): Install Web Modeler with a non-public PostgreSQL schema to comply with central IT requirements..

Custom JWKS and JWT Algorithms Support (REST API) (SM only): Self-managed customers now have full control over JWT configurations for enhanced security and compatibility.


Simplified Deployment  (SM only): Authenticate deployments with your existing user tokens, rather than entering shared secrets.. This further simplifies the deployment process beyond our 8.6 release.

Play Multi-Tenancy Support (SM only): Use your existing development cluster with tenancy so multiple teams can develop on the same cluster without impacting each other.